### DISCUSSION TITLE
Green LED at GPIO4

### DISCUSSION THREAD

**Discussion by akula3** [klaasnicolaas/home-assistant-glow#177] [Q&A] [answered]
What is it for?

**Comment by klaasnicolaas** [comment, marked as answer]
What would you like to know? We use an LED where the 3 colors (red, green, blue) each need their own GPIO pin.

**Reply by akula3** [reply]
What is the function of the green LED? When is it on/flashing? According to YAML, it is just connected to GPIO Pin without any action on that pin.

**Reply by klaasnicolaas** [reply]
Indeed, as you've seen in the code, it currently has no function.
